_episodes/01-introduction.md

```diff
@@ -88,7 +88,7 @@ The following could technically be achieved using Deep Learning, but it would pr
 
 > ## Deep Learning Problems Exercise
 >
-> Which of the following could you apply Deep Learning to?
+> Which of the following should you apply Deep Learning to?
```

Comment:
```suggestion
> Which of the following would you apply Deep Learning to?
```

Reply:
@dafnevk I am commenting on my own PR. github offers too many options to change my own code. :-)